src/cmap/command_monitoring_events.ts

```diff
@@ -65,6 +72,7 @@ export class CommandSucceededEvent {
   commandName: string;
   reply: unknown;
   serviceId?: ObjectId;
+  name: string = COMMAND_SUCCEEDED;
```

Comment:
```suggestion
  name = COMMAND_SUCCEEDED;
```

If we leave out the string annotation then this gets to be a string literal type, which could be used for type narrowing, same for the rest.